inc/sitemaps/class-sitemaps-cache.php

```diff
@@ -41,6 +47,190 @@ public function is_enabled() {
 	}
 
 	/**
+	 * Get the cache key for a certain type and page
+	 *
+	 * Example key format for type "post", page 1: wpseo_sitemap_post_1:akfw3e_23azBa
+	 *
+	 * @param null|string $type The type to get the key for. Null or self::SITEMAP_INDEX_TYPE for index cache.
```

Comment:
This doc is inaccurate. We should describe that you can pass a certain type here, such as `video`.